Navitia.php

```diff
@@ -2,93 +2,78 @@
 
 namespace CanalTP\NavitiaPhp;
 
-use Symfony\Component\Yaml\Yaml;
-use Symfony\Component\OptionsResolver\OptionsResolver;
 use GuzzleHttp\Client;
 use GuzzleHttp\Exception\ClientException;
```

Comment:
After reflexion, I would had an alias : "as GuzzleClientException" to more clear in code